twistlock/datadog_checks/twistlock/data/conf.yaml.example

```diff
@@ -0,0 +1,24 @@
+init_config:
+
+instances:
+  ## @param endpoint - string - required
+  ## The twistlock url
+  #
+  - url: 'http://localhost:8081'
+
+  ## @param username - string - required
+  ## The twistlock prometheus endpoint
+  #
+    username: '<USERNAME>'
+
+  ## @param password - string - required
+  ## The twistlock prometheus endpoint
+  #
+    password: '<PASSWORD>'
+
```

Comment:
Shouldn't this have `tags` and `ssl_verify` ?

Reply:
yes, I'll add them